design.py

```diff
@@ -1,40 +1,94 @@
 import sys
-import json
-#prompt a user to create a recipe
-#Ask for Ingredients
-#Ask for Instructions
+
 #Save/Store it into dictionary
 def create():
-    ask_recipe = ''
-    ask_instruction = ''
-    ask_ingredients = ''
-    recipe_dict = dict()
-    #counter = str(1)
-    print("Create a recipe! Type 'exit' to leave")
-    for ask_recipe in range (1):
-        ask_recipe = input("Recipe Name: ")
-        recipe_dict.update(ask_recipe = 'Nothing Yet')
-        if ask_recipe == 'exit':
-            exit(1)
-
-    while ask_instruction != 'exit':
-        ask_instruction = input("Instructions: ")
-        if ask_instruction == 'exit':
-            while ask_ingredients != 'exit':
-                ask_ingredients = input("Ingredients: ")  
-                if ask_ingredients == 'exit':
-                    #recipe_dict.update(ask_recipe = ask_instruction)
-                    #recipe_dict.update(ask_recipe, ask_ingredients)
-                    for key, value in recipe_dict.items():
-                        print(key,':',value)
-                        exit(1)
-
-
-
-    print("Create a recipe! Enter '0' to exit")
-    print(ask_recipe)
-    print(ask_instruction)
-    print(ask_ingredients)
+    
+    recipe_name = ''
+    recipe_dict = {}
+
+    print("Create a recipe!")
+    recipe_name = input("Recipe Name: ")
+
+    ingredientList = []
+    ingredients = ''
+
+    while True:
+        ingredients = input("Ingredients: ")  
+        if ingredients == 'exit':
+            break
+        ingredientList.append(ingredients)
+
+
+    instructionList = []
+    instructions = ""
+
+    while True:
+        instructions = input("Instructions: ")
+        if instructions == 'exit':
+            break
+        instructionList.append(instructions)
+
+    class ingr_instr_list:
+        ingrSave = ingredientList
```

Comment:
Classes have nothing to do with the fact that we're using them as a way to easily save things. If we go back to the example this is the equivalent of naming a property in our Animal class to `has_horns_save= boolean`.

So when you want to find out if an Animal has horns you would have to ask the program by saying `someAnimal.has_horns_save` instead of a more english readable `someAnimal.has_horns`.

Remember that classes are objects so you should name the properties of the object just like you would in real like. If we were writing down what properties a phone has in real life we wouldn't do `total_cameras_save` we would simply just say `total_cameras`.

Once we have the object all filled out then saving it we can do just because its in one nice package. Just like you can save a string or save an integer.